driver-core/src/test/functional/com/mongodb/internal/async/client/ChangeStreamProseTest.java

```diff
@@ -102,7 +101,7 @@ public void testResumeOneTimeOnError() {
         AsyncBatchCursor<ChangeStreamDocument<Document>> cursor = createChangeStreamCursor();
 
         insertOneDocument();
-        setFailPoint("getMore", 10107);
+        setFailPoint(new FailPoint("getMore", 10107));
```

Comment:
I think this change was from a comment from @divjotarora.  I don't see the conversation anymore, but i don't find this to be any clearer this way.